src-cpp-structure/projects/ServicesTest/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
+
+project(Status.ServicesTest DESCRIPTION "Test Status Services Project")
+
+SET(STATUS_RCC ${CMAKE_SOURCE_DIR}/../resources.rcc)
+SET(STATUS_QRC ${CMAKE_SOURCE_DIR}/../ui/resources.qrc)
+SET(STATUS_RESOURCES_QRC ${CMAKE_SOURCE_DIR}/../resources/resources.qrc)
+
+conan_basic_setup(KEEP_RPATHS)
+
+add_executable(${PROJECT_NAME})
+
+add_dependencies(${PROJECT_NAME} Status.Services)
+
+target_link_libraries(
+    ${PROJECT_NAME}
+    Qt5::Core
+    ${CONAN_LIBS_GTEST}
```

Comment:
That's bad practice. `target_link_libraries` should be package manager agnostic, the dependencies should be found through `find_package`, see: https://blog.conan.io/2021/10/19/New-CMake-Transparent-Integration-Guide.html

Long story short: global vars approach will be deprecated and removed with Conan 2.0, we should use `CMakeToolchain` + `cmake_find_package` and `cmake_find_package_multi` generators for now, see: https://github.com/status-im/status-desktop/blob/master/conanfile.py